File: LeetCode/Javascript/Easy/TwoSum.js
// https://leetcode.com/problems/two-sum/description/

const twoSum = function(nums, target) {
  for (let i = 0; i < nums.length; i++) {
    const currentNumber = nums[i];

    for (let j = i + 1; j < nums.length; j++) {
      const nextNumber = nums[j];

      if (currentNumber + nextNumber === target && i !== j) {
        return [i, j];
      }
    }
  }
};

[2,7,11,15]

File: LeetCode/Python/Easy/TwoSum.py
# Problem link: https://leetcode.com/problems/two-sum/

class Solution:
  def twoSum(self, nums: List[int], target: int) -> List[int]:
    for index in range(0, len(nums)):
      currentNumber = nums[index]

      for index2 in range(index + 1, len(nums)):
        nextNumber = nums[index2]

        if currentNumber + nextNumber == target:
          return [index, index2]
